src/_components/form-elements/form-elements.css

```diff
@@ -17,46 +17,6 @@
   margin-top: 25px;
 }
 
-.form-input {
```

Comment:
Det her kommer ikke til at ødelægge noget på eksisterende formularer?

Reply:
Jo, godt input. De eksisterende elementer er ikke wrappet i form-input-container.
Er løsningen ikke bare at revert den ændring og lade .form-input være for nu, så de nuværende elementer ligner det de gør dagen i dag? :)

Reply:
jo, det vil jeg mene